### PR TITLE
Fix all compilation warnings and remove deprecated methods

### PR DESCRIPTION
**Pull request by adjamhub**

### CODE REVIEW

**Comment by tsujan** [comment]
Was in conflict with https://github.com/lxqt/lximage-qt/pull/397. Please rebase or make another PR.

**Comment by adjamhub** [comment]
Rebased!
The conflicts were due to similar fixes with deprecated methods :)

File: src/imageview.cpp [hunk 1]
```diff
@@ -98,7 +98,9 @@ void ImageView::onFileDropped(const QString file) {
 }
 
 void ImageView::wheelEvent(QWheelEvent* event) {
-  int delta = event->delta();
+  QPoint angleDelta = event->angleDelta();
+  Qt::Orientation orient = (qAbs(angleDelta.x()) > qAbs(angleDelta.y()) ? Qt::Horizontal : Qt::Vertical);
+  int delta = (orient == Qt::Horizontal ? angleDelta.x() : angleDelta.y());
```
Comment (tsujan) [inline src/imageview.cpp:103]: I think Qt should restore `QWheelEvent::delta()` or add another method similar to it. Each time we need delta, we should do this calculation to get rid of warnings :(

**Review by tsujan — APPROVED** [review]
GTM.

Thanks!

File: src/imageview.cpp (hunk 1 above)
Comment (adjamhub) [reply src/imageview.cpp:103]: Definitely!
I can try asking in some Qt chats about...

**Pull request merged**